ansible/localhost/base/tasks/main.yml

```diff
@@ -21,7 +21,79 @@
         - python-psycopg2
         - git
         - ntp
+        - apache2
+        - apache2.2-common
+        - apache2-mpm-prefork
+        - apache2-utils
+        - openssl
+        - ssl-cert
+        - libexpat1
+        - libapache2-mod-wsgi
 
     - name: Check NTP service to make sure it's running
       service: name=ntp state=started enabled=yes
-      sudo: yes
\ No newline at end of file
+      sudo: yes
+
+    - name: Check Apache service to make sure it's running
+      service: name=apache2 state=started enabled=yes
+      sudo: yes
+
+    - name: enabled mod_rewrite
+      apache2_module: name=rewrite state=present
+      notify:
+        - restart apache2
+
+    - name: a2ensite ssl
+      command: a2enmod ssl
+      notify:
+        - restart apache2
+
+    - name: check for SSL folder
+      stat: path=/etc/apache2/ssl
+      register: ssl_path
+
+    - name: create SSL folder
+      command: mkdir /etc/apache2/ssl
+      when: ssl_path.stat.exists == false
+
+    - name: create self-signed SSL cert
+      command: openssl req -new -nodes -x509 -subj "/C=US/ST=Massachusetts/L=Boston/O=Carb/CN={{ domain }}" -days 3650 -keyout /etc/apache2/ssl/server.key -out /etc/apache2/ssl/server.crt -extensions v3_ca creates=/etc/apache2/ssl/server.crt
```

Comment:
Can we not use the snakeoil cert that comes with Ubuntu? I don't like the idea of having a "Carb" self-signed cert in an open source project.